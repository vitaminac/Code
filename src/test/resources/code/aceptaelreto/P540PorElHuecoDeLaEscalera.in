2
7 18 6 5
10 10 10 0
