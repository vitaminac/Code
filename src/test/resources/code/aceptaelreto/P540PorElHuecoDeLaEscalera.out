113 239
100 200
